### PR TITLE
Conan commands docstrings

### PR DESCRIPTION
**Pull request by franramirez688**
Changelog: Fix: Unifying Conan commands help messages.
Docs: omit

### CODE REVIEW

File: conan/cli/commands/cache.py [hunk 1]
```diff
@@ -9,19 +9,20 @@
...
+    Perform file operations in the local cache (of recipes and/or packages).
     """
     pass
 
 
 @conan_subcommand(formatters={"text": default_text_formatter})
 def cache_path(conan_api: ConanAPI, parser, subparser, *args):
     """
-        Shows the path in the Conan cache af a given reference
+    Show the path to the Conan cache af a given reference.
```
Comment (franramirez688) [inline conan/cli/commands/cache.py]:
```suggestion
    Show the path to the Conan cache if a given reference.
```

File: conan/cli/commands/cache.py [hunk 2]
```diff
@@ -9,19 +9,20 @@
...
+    Perform file operations in the local cache (of recipes and/or packages).
     """
     pass
 
 
 @conan_subcommand(formatters={"text": default_text_formatter})
 def cache_path(conan_api: ConanAPI, parser, subparser, *args):
     """
-        Shows the path in the Conan cache af a given reference
+    Show the path to the Conan cache if a given reference.
```
Comment (memsharded) [inline conan/cli/commands/cache.py]:
```suggestion
    Show the path to the Conan cache for a given reference.
```

File: conan/cli/commands/cache.py [hunk 3]
```diff
@@ -54,22 +55,23 @@ def cache_path(conan_api: ConanAPI, parser, subparser, *args):
...
                            help="Clean download folders")
     subparser.add_argument('-p', '--package-query', action=OnceArgument,
-                           help="Remove all packages (empty) or provide a query: "
+                           help="Remove only the packages matching a specific query, e.g., "
                                 "os=Windows AND (arch=x86 OR compiler=gcc)")
     args = parser.parse_args(*args)
 
     if not args.source and not args.build and not args.download:
-        raise ConanException("Define at least one argument from --source, --build, --download")
+        raise ConanException("Define at least one argument between [--source, --build, --download]")
```
Comment (memsharded) [inline conan/cli/commands/cache.py]:
```suggestion
        raise ConanException("Define at least one argument among [--source, --build, --download]")
```

not sure about this, but I'd say between not great either

File: conan/cli/commands/create.py [hunk 4]
```diff
@@ -23,13 +23,13 @@ def json_create(deps_graph):
...
     """
-    Create a package
+    Create a package.
     """
     common_args_export(parser)
     add_lockfile_args(parser)
     add_common_install_arguments(parser)
     parser.add_argument("--build-require", action='store_true', default=False,
-                        help='The provided reference is a build-require')
+                        help='If the provided reference is a build-require')
```
Comment (memsharded) [inline conan/cli/commands/create.py]: Not sure about this, this is not a conditional. It tells that the reference is a build-require.

File: conan/cli/commands/graph.py [hunk 5]
```diff
@@ -16,7 +16,7 @@
 @conan_command(group="Consumer")
 def graph(conan_api, parser, *args):
     """
-    Computes a dependency graph, without  installing or building the binaries
+    Compute a dependency graph, without  installing or building the binaries.
```
Comment (memsharded) [inline conan/cli/commands/graph.py]: I am not sure the docstrings should be plural/first person.

The command ``conan graph`` **computes** a dependency graph...

**Review by memsharded — COMMENTED** [review]
Some minor comments, that might require a more native English speaker to clarify.

File: conan/cli/commands/cache.py (hunk 3 above)
Comment (AbrilRBS) [reply conan/cli/commands/cache.py]: Among, as between is for when discussing two items 👍

File: conan/cli/commands/create.py (hunk 4 above)
Comment (AbrilRBS) [reply conan/cli/commands/create.py]: Agree. `Whether the provided reference is a build-require` sounds nicer to me

File: conan/cli/commands/graph.py (hunk 5 above)
Comment (AbrilRBS) [reply conan/cli/commands/graph.py]: Python's stdlib uses the imperative tense, such as:

```py
def isinstance(x, A_tuple):
    """
    Return whether an object is an instance of a class or of a subclass thereof.
    """
    ...
```

I'm happy to go with whatever you prefer, no strong opinions either way

File: conan/cli/commands/cache.py [hunk 6]
```diff
@@ -9,19 +9,20 @@
...
 @conan_subcommand(formatters={"text": default_text_formatter})
 def cache_path(conan_api: ConanAPI, parser, subparser, *args):
     """
-        Shows the path in the Conan cache af a given reference
+    Show the path to the Conan cache for a given reference.
     """
     subparser.add_argument("reference", help="Recipe reference or Package reference")
     subparser.add_argument("--folder", choices=['export_source', 'source', 'build'],
-                           help="Show the path to the specified element. The 'build'"
+                           help="Shows the path to the specified element. The 'build'"
```
Comment (czoido) [inline conan/cli/commands/cache.py]:
```suggestion
                           help="Path to show. The 'build'"
```

**Pull request merged**